packages/tooling/adaptive-ui-explorer/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@microsoft/adaptive-ui-explorer",
+  "version": "1.0.0-alpha.1",
+  "description": "A playground for Adaptive UI",
+  "type": "module",
+  "main": "dist/esm/index.js",
+  "types": "dist/adaptive-ui-explorer.d.ts",
+  "unpkg": "dist/esm/index.js",
```

Comment:
This package should declare `"private": true` so it doesn't get published to npm

Reply:
We should have a conversation about this, but at some point I would like to publish it. It's built as a set of components people building on FAST and Adaptive UI can use to show off _their_ design system. For instance, For the Color Explorer site we host, we'd pull in the components from this package, Fluent UI web components, and configure the SampleApp and SamplePage (discussed below) to use "fluent-*".

It has a little way to go before it's fully usable for this, so I'd be fine keeping it private for now though.

Reply:
Thats fine. We can absolutely plan the long tail of this package, but we don't want to block package publishing or publish it accidently before it's ready, so it needs to be private for now.